File: new/atoms/sup-keyed-cols.test.mjs
import * as m from './sup-keyed-cols.mjs';
const SNAP = {"SUP_KEYED_COLS":"[\"supporters\",\"events\"]"};
let f = 0;
for (const [k, s] of Object.entries(SNAP)) if (JSON.stringify(m[k]) !== s) { console.error('✗ ' + k + ' סטה מהצילום'); f = 1; }
if (f) process.exit(1); console.log('✓ sup-keyed-cols: צילום-ערך תואם — ירוק');

File: new/dart-maor/sup-keyed-cols_test.dart
// בדיקת-חוזה (רתמת-זהב) · sup-keyed-cols — צילום-ערך jsonEncode≡JSON.stringify.
// חוללה מכנית (gen-data-dart). הרצה: dart run --enable-asserts ⇒ OK
import 'dart:convert';
import 'sup-keyed-cols.dart';

void main() {
  // ‏1) צילום-ערך: jsonEncode ≡ JSON.stringify של המקור (זהה-ביט).
  if (jsonEncode(supKeyedCols) != '["supporters","events"]') {
    throw StateError('FAIL supKeyedCols: צילום-הערך סטה');
  }
  print('OK sup-keyed-cols: 1 data-snapshot(s) passed');
}
